checklists/azure_arc_checklist.en.json

```diff
@@ -212,6 +212,7 @@
             "text": "Design a monitoring strategy to send metrics and logs to an Log Analytics workspace",
             "guid": "74d1102c-ac6a-4ae0-8e6a-84de5df47d2d",
             "severity": "Medium",
+            "graph": "resources | where type =~ 'microsoft.hybridcompute/machines' | project id, arcMachineName = name | join kind=leftouter (resources | where type == 'microsoft.hybridcompute/machines/extensions' | where name contains 'azuremonitor' | extend baseIdParts = split(id, '/extensions/') | extend extensionMachineId = tostring(baseIdParts[0]) | project extensionMachineId, extensionId = id) on $left.id == $right.extensionMachineId | extend compliant = iif(extensionMachineId == '', 'No', 'Yes') | project id, compliant",
```

Comment:
* `compliant` should be a boolean. You could do something like this: `| extend compliant = isnotempty(extensionMachineId)` (see https://learn.microsoft.com/en-us/kusto/query/isnotempty-function?view=microsoft-fabric for more info on the `isnotempty` function)

Reply:
I believe I made the change you requested. Thanks @erjosito!